gensim/corpora/wikicorpus.py

```diff
@@ -365,6 +426,13 @@ def extract_pages(f, filter_namespaces=False):
                 if ns not in filter_namespaces:
                     text = None
 
+            if filter_articles is not None:
+                if not filter_articles(elem, namespace=namespace, title=title,
+                                       text=text, page_tag=page_tag,
```

Comment:
We use hanging indent in Gensim (not vertical alignment): https://www.python.org/dev/peps/pep-0008/#id3

Why? Makes code easier to maintain (people forget to re-indent after modifying), and some people (me!) also find it more consistent and easier to read :)